source/app/iris_engine/module_handler/module_handler.py

```
@@ -465,10 +464,10 @@ def task_hook_wrapper(self, module_name, hook_name, hook_ui_name, data, init_use
             _obj.append(_obj_a)
 
     except Exception as e:
-        log.exception(e)
+        logger.exception(e)
         raise Exception(e)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)